webroot/src/models/User/User.model.spec.ts

```diff
@@ -34,24 +82,184 @@ describe('User model', () => {
         expect(user.email).to.equal(data.email);
         expect(user.firstName).to.equal(data.firstName);
         expect(user.lastName).to.equal(data.lastName);
+        expect(user.permissions).to.matchPattern([
+            {
+                compact: new Compact({ type: CompactType.ASLP }),
+                isRead: true,
+                isAdmin: true,
+                states: [
+                    {
+                        isWrite: true,
+                        isAdmin: true,
+                        '...': '',
```

Comment:
What's this mean?

Reply:
It's the chai matchPattern shorthand for noting that there are other properties. In this case, chai had a little trouble supporting the terse / readable version of this test. Since the State stuff is well-tested elsewhere, I just decided to check the User-related permission stuff here and not hack into the chai plugin.

Reply:
got it!